### PR TITLE
add pages for caja to make file

### PR DESCRIPTION
**Pull request by NiceandGently**

### CODE REVIEW

**Comment by monsta** [comment]
@flexiondotorg: works perfectly in Debian Testing, I get help page for Caja and even the backup stuff is mentioned on it :)

**Comment by flexiondotorg** [comment]
Closes #403

**Comment by dcharlespyle** [comment]
Tried to build the package using this commit and get the following:

CDPATH="${ZSH_VERSION+.}:" && cd . && /bin/sh /home/dcpyle/rpmbuild/BUILD/mate-user-guide-1.10.0/missing aclocal-1.14 
CDPATH="${ZSH_VERSION+.}:" && cd . && /bin/sh /home/dcpyle/rpmbuild/BUILD/mate-user-guide-1.10.0/missing autoconf
 cd . && /bin/sh /home/dcpyle/rpmbuild/BUILD/mate-user-guide-1.10.0/missing automake-1.14 --gnu
/home/dcpyle/rpmbuild/BUILD/mate-user-guide-1.10.0/missing: line 81: automake-1.14: command not found
WARNING: 'automake-1.14' is missing on your system.
         You should only need it if you modified 'Makefile.am' or
         'configure.ac' or m4 files included by 'configure.ac'.
         The 'automake' program is part of the GNU Automake package:
         <http://www.gnu.org/software/automake>
         It also requires GNU Autoconf, GNU m4 and Perl in order to run:
         <http://www.gnu.org/software/autoconf>
         <http://www.gnu.org/software/m4/>
         <http://www.perl.org/>
Makefile:355: recipe for target 'Makefile.in' failed
make: *** [Makefile.in] Error 1
make: *** Waiting for unfinished jobs....
error: Bad exit status from /var/tmp/rpm-tmp.bJZD64 (%build)


RPM build errors:
    Bad exit status from /var/tmp/rpm-tmp.bJZD64 (%build)

I checked and aclocal is installed and up-to-date.  Tried to do a few other things to try to get this update and nothing is working.  I get errors for missing files.  Trying to copy them manually into the source archive using engrampa gets me errors while copying and four of the files copied never appear in the archive.  Several, repeated attempts to copy the files resulted in a broken archive that would not build.  Not sure what is going on but I am trying to do this in F22.

**Comment by dcharlespyle** [comment]
Never mind.  I redownloaded the files and this time did not use Engrampa to work with the archive files.  I then installed the package and now still see the following:

Page Not Found
The page ‘goscaja-1’ was not found in the document ‘help:mate-user-guide’.

I have never had failures like this before on this system.  So, not sure what is going on at the moment.  Maybe I just need to reboot.

**Comment by NiceandGently** [comment]
try 'autoreconf -fi' before ececuting ./configure

you need the 2 latest commits from nautilus
https://github.com/mate-desktop/caja/commits/master

**Comment by dcharlespyle** [comment]
I did use the latest commits.  But, I will try the command you mentioned and see what that gets me.  Thanks.

**Comment by NiceandGently** [comment]
for changes with caja is a session restart needed, or 'killall caja'

**Comment by dcharlespyle** [comment]
Did that, too.  Now I get the following while trying to build packages.

ERROR   0001: file '/usr/libexec/caja-convert-metadata' contains a standard rpath '/usr/lib64' in [/usr/lib64]
ERROR   0001: file '/usr/bin/caja-autorun-software' contains a standard rpath '/usr/lib64' in [/usr/lib64]
ERROR   0001: file '/usr/bin/caja-file-management-properties' contains a standard rpath '/usr/lib64' in [/usr/lib64]
ERROR   0001: file '/usr/bin/caja' contains a standard rpath '/usr/lib64' in [/usr/lib64]
ERROR   0001: file '/usr/bin/caja-connect-server' contains a standard rpath '/usr/lib64' in [/usr/lib64]
error: Bad exit status from /var/tmp/rpm-tmp.lrMcIl (%install)


RPM build errors:
    Bad exit status from /var/tmp/rpm-tmp.lrMcIl (%install)

**Comment by dcharlespyle** [comment]
OK. Found that if I even used Engrampa once on any file associated with this project, I get errors.  I redownloaded all files and used GNOME Shell's Archive Manager instead.  I was able to build everything just fine.  But, now I see the following when I access it from the Help menu in an open Caja window:

screenshot-help

**Comment by NiceandGently** [comment]
i've no clue what happens, but maybe you prefer some pre-released rpm's :)
https://dl.dropboxusercontent.com/u/49862637/Mate-desktop/Bugs/rpm.tar.xz

**Comment by dcharlespyle** [comment]
I'm getting to that point, yes. :)

Not sure what is going on.  I've been building packages all day and these ones are the only ones giving me issues (well, OK, those and and one package from beesu that GNOME devs broke a couple versions ago with changes to gedit).  Thank you for the link.  I'll try those momentarily.

**Comment by dcharlespyle** [comment]
Excellent. All is well and no restarting of Caja was required.  Another oddity is that mate-user-guide was being said to not be installed but an rpm -qa showed that it was.  Dnf was having a hissy-fit yesterday, so now I wonder if there are bugs in dnf and engrampa that resulted in my having issues.  Thanks for the updated files.  At least I still will have some hair left. :)

**Comment by NiceandGently** [comment]
ps, help:mate-user-guide/files is the right link.
Means, caja works well. In /usr/share/help/C/mate-user-guide should be a fle called files.page .

**Comment by dcharlespyle** [comment]
It now goes to that page.  Unfortunately, there still are a couple links that do the following:

screenshot-help-1

screenshot-help-2

**Comment by NiceandGently** [comment]
LOL,OK,
please give us some time to fix that , smilyglasses

**Comment by NiceandGently** [comment]
@monsta 
Now i know why this wasn't enabled :)

**Comment by monsta** [comment]
Ok, I've searched for these... This time the wrong links aren't in Caja code, they're right here in m-u-g :)

**Comment by NiceandGently** [comment]
@dcharlespyle 
https://github.com/mate-desktop/mate-user-guide/commit/81109ec
Can you please test this?
You can simply copy the new and updated files to /usr/share/help/C/mate-user-guide/

PS: some images are missing ;)

**Comment by dcharlespyle** [comment]
So far, so good, except for on the page "Browse files on a server or network share" there are two broken links.  This is what I see when I click either link "URL" or "listed below" on that page:

screenshot-browse files on a server or network share

**Comment by monsta** [comment]
Hmm, these two links work fine for me in Debian Testing. They take me down the page as they should.

**Comment by dcharlespyle** [comment]
The links now do what you report.  Not sure what is going on.  At first, neither worked.  Then, one worked and the other didn't.  Now, both work.  So, my thanks for fixing it.  I guess I just needed to wait for it to start working.  Maybe an update had something to do with it? I have updates-testing enabled, so I get updates almost every day.

**Comment by monsta** [comment]
Might be yelp misbehaving...

**Comment by dcharlespyle** [comment]
Could be.  I seem to remember an update for yelp came up recently.

**Comment by NiceandGently** [comment]
@dcharlespyle 
i updated caja and mate-user-guide for fedora
https://admin.fedoraproject.org/updates/mate-user-guide-1.10.1-0.1.git20150618.61aec06.fc22,caja-1.10.1-2.fc22?_csrf_token=6d160bbc625246b6e9b79a5f6318fec87b4c7540
Please test.

**Comment by dcharlespyle** [comment]
Tested and got this:

screenshot-help

**Comment by NiceandGently** [comment]
where did you click on it?

**Comment by NiceandGently** [comment]
@monsta 
@flexiondotorg 
Could you you please also test caja help?
m-u-g is not really my project.

**Comment by dcharlespyle** [comment]
I clicked it from here:

screenshot-quickly preview music-sound files

**Comment by dcharlespyle** [comment]
I also found another one while clicking help in a Panel's Launcher Properties dialog.  I got the following there:

screenshot-help

**Comment by monsta** [comment]
caja-preview link is here in m-u-g; panel help link is in mate-panel - but there's no panel help in m-u-g at all, so this is not fixable right now.

Anyway, please open new issue(s), this discussion has become too long and hard to follow.

**Comment by NiceandGently** [comment]
https://github.com/mate-desktop/mate-user-guide/pull/6
